tests/Functional/Schema/MySQLSchemaManagerTest.php

```diff
@@ -533,19 +533,12 @@ public function testEnsureTableWithoutOptionsAreReflectedInMetadata(): void
         $this->connection->executeStatement('DROP TABLE IF EXISTS test_table_empty_metadata');
 
         $this->connection->executeStatement('CREATE TABLE test_table_empty_metadata(col1 INT NOT NULL)');
-        $onlineTable = $this->schemaManager->listTableDetails('test_table_empty_metadata');
+        $onlineTable = $this->schemaManager->getTable('test_table_empty_metadata');
 
         self::assertNotEmpty($onlineTable->getOption('engine'));
         // collation could be set to default or not set, information_schema indicate a possibly null value
         self::assertFalse($onlineTable->hasOption('autoincrement'));
         self::assertEquals('', $onlineTable->getOption('comment'));
         self::assertEquals([], $onlineTable->getOption('create_options'));
     }
-
-    public function testParseNullCreateOptions(): void
-    {
-        $table = $this->schemaManager->listTableDetails('sys.processlist');
-
-        self::assertEquals([], $table->getOption('create_options'));
-    }
```

Comment:
The migration of the test suite to the new API renders this test invalid. Originally (https://github.com/doctrine/dbal/pull/3412), it was introduced to test the create options of a view but neither `listTableDetails()` nor `getTable()` return the view definition. The former returns a non-existing table and the latter throws an exception.

I don't know how to test this case properly and whether it's valid at all (`INFORMATION_SCHEMA.TABLES` contains NULL create options only for views). The original issue (https://github.com/doctrine/dbal/issues/3410) doesn't mention the steps to create such a table. I would not revert the fix itself since the NULL value of create options is still possible.